wormhole-connect/src/views/v2/TxHistory/Widget/Item.tsx

```diff
@@ -84,12 +78,14 @@ const WidgetItem = (props: Props) => {
     onExpire: () => setEtaExpired(true),
   });
 
-  const { isCompleted } = useTrackTransferInProgress({
+  const { isCompleted, isReadyToClaim } = useTrackTransferInProgress({
     eta,
     receipt,
     route,
   });
 
+  const { isLoading: isLoadingResume } = useResumeTransaction(resumeTxLocal);
```

Comment:
You're not reading the `error` return value - should we do that and handle errors somehow?

Reply:
Maybe we can show this as a alert banner underneath the widget? Let me get some visuals for that.

Reply:
cool. yeah error states for little things like this can be tricky to get right design-wise but it's what separates robust products from janky ones <3